package.json

```diff
@@ -33,18 +33,21 @@
     }
   ],
   "bin": "cli.js",
+  "typings": "types/generated.d.ts",
   "engines": {
     "node": ">=0.10.0"
   },
   "scripts": {
     "test": "xo && nyc --cache --reporter=lcov --reporter=text tap --no-cov --timeout=150 test/*.js test/reporters/*.js",
     "test-win": "tap --no-cov --reporter=classic --timeout=150 test/*.js test/reporters/*.js",
-    "visual": "node test/visual/run-visual-tests.js"
+    "visual": "node test/visual/run-visual-tests.js",
+    "prepublish": "npm run make-ts",
+    "make-ts": "node -e \"require('babel-register')({ presets: 'babel-preset-es2015' }); require('./types/make');\""
```

Comment:
Actually, we could just use `babel-node` here instead of `node`. We just need to add `babel-cli` as a devDependency. Then we could drop this boilerplate.

https://babeljs.io/docs/usage/cli/#babel-node